src/components/sidebar/linkView/LinkView.tsx

```diff
@@ -170,37 +166,31 @@ class LinkView extends React.Component<ILinkViewProps, ILinkViewState> {
                     <div className={classnames(s.subTopic)}>
                         KUTSU-/JÄTTÖ-/OTTOP
                     </div>
-                    <div>
-                        <Dropdown
-                            onChange={this.onChange}
-                            items={['0 - Ei', '1 - Ei', '2 - Ei']}
-                            selected={'0 - Ei'}
-                        />
-                    </div>
+                    <Dropdown
+                        onChange={this.onChange}
+                        items={['0 - Ei', '1 - Ei', '2 - Ei']}
+                        selected={'0 - Ei'}
+                    />
                 </div>
                 <div className={s.inputContainer}>
                     <div className={classnames(s.subTopic)}>
                         AJANTASAUSPYSÄKKI
                     </div>
-                    <div>
-                        <Dropdown
-                            onChange={this.onChange}
-                            items={['Kyllä']}
-                            selected={'Ei'}
-                        />
-                    </div>
+                    <Dropdown
+                        onChange={this.onChange}
+                        items={['Kyllä']}
+                        selected={'Ei'}
```

Comment:
How can 'Ei' be selected if items doesn't contain it?

Reply:
This isn't the final solution yet, only a static page. And this is how it was before